src/main/io/serial.c

```diff
@@ -459,7 +459,17 @@ void serialInit(bool softserialEnabled, serialPortIdentifier_e serialPortToDisab
             }
         }
 
-        if ((serialPortUsageList[index].identifier == SERIAL_PORT_SOFTSERIAL1
+#ifndef SITL
```

Comment:
I know we're not being consistent with it, but I think making code outside of `src/main/target` independent of target names is a good thing, and this should be `SIMULATOR_BUILD`.

Reply:
Ah, no wonder I couldn't find that numerous `#ifdef SITL` BS in the repo anymore!

Reply:
Will change these as part of a target name define cleanup.

Reply:
👍

Reply:
Done in #5877.